packages/vue/src/components/setup-totp.vue

```diff
@@ -18,15 +18,13 @@ const logger = new Logger('SetupTotp-logger');
 
 // `facade` is manually typed to `UseAuthenticator` for temporary type safety.
 const facade: UseAuthenticator = useAuthenticator();
-const { updateForm, submitForm, toSignIn } = facade;
-const { error, isPending, QRFields, totpSecretCode, username } = toRefs(facade);
```

Comment:
`QRFields`, `totpSecretCode`, `username` are static values and do not require reactivity